SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
     >
-    > If you ever need to run with Unreal networking for workflow or validation reasons, untick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config files under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
-    >
-    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in pre-alpha, switching back to Unreal's networking may be a viable way to work around some issues you may encounter, and may speed up your iteration. However, in Unreal networking mode you lose access to key multi-server features of the GDK, which may lead to erratic behavior.
+    > If you ever need to run with default Unreal networking for workflow or validation reasons, un-tick `Spatial Networking` in the `Play` drop down menu in the Unreal editor. This flag is maintained in your Unreal config file `<GameRoot>/Config/DefaultGame.ini` under `/Script/EngineSettings.GeneralProjectSettings` and is also valid for non-editor builds.
     >
-    > In the future, we expect to achieve parity with Unreal networking in terms of iteration speed and stability, thus removing the need for a networking mode switch.
-1. Open the game's uproject file in the Editor and navigate to *Edit > Project Settings... > SpatialOS Unreal GDK > Toolbar*.
+    > You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in alpha, switching back to Unreal's networking may be a viable way for debugging any issues you may be having and may speed up your iteration times. However, in Unreal default networking mode you lose access to the multi-server features of the GDK, which may lead to erratic behavior.
```

Comment:
Do we want a screenshot for the check box?

Reply:
I have suggested standarising the menu instructions to the way we always write them, which should make it clearer.

We could link to the toolbar page - although I note that this is not in the toolbar page. 
https://docs.improbable.io/unreal/pre-alpha/content/toolbar

Reply:
Suggest some edits:

"You can easily switch between networking modes using this checkbox. As the SpatialOS Unreal GDK is in alpha, switching back to Unreal's networking may be a viable way for debugging any issues you may be having and may speed up your iteration times. However, in Unreal default networking mode you lose access to the multi-server features of the GDK, which may lead to erratic behavior."

* "You can easily switch between networking modes using this checkbox." <- don't need this - it's clear from how we have written it above.
* SpatialOS Unreal GDK -> the GDK
* may be a viable way -> "useful" probs better as this suggests that SpatialOS networking may be not viable and we don't want to suggest that, I think. :-) 
* any issues you may be having and may speed up your iteration times -> complex use of present continuous passive which we don't need. "iteration" on its own means going round in circles.
* erratic behavior probs needs more attention - make this into a warning? 
* we use two different terms for Unreal's default networking - let's use a consistent term to be super clear and avoid any mental trips.
->
**Warning: **
As the GDK is in alpha, switching back to Unreal default networking mode can be a useful way to debug and so speed up your development iteration. However, you lose access to the multi-server features of the GDK in Unreal default networking mode which may lead to erratic behavior.

Reply:
Added.